File: Bài 8/AutoCompleteTextView/app/src/main/java/com/example/autocompletetextview/MainActivity.java
package com.example.autocompletetextview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.ArrayAdapter;
import android.widget.AutoCompleteTextView;

import java.util.ArrayList;
import java.util.List;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        AutoCompleteTextView autoCompleteTextView = findViewById(R.id.autoCompleteTV);

        String[] countries = getResources().getStringArray(R.array.Countries);
        CountryAdapter countryAdapter = new CountryAdapter(this, R.layout.layout_custom, getListContries());
        autoCompleteTextView.setAdapter(countryAdapter);
    }

    private List<Country> getListContries() {
        List<Country> list = new ArrayList<>();
        list.add(new Country("VietNam"));
        list.add(new Country("Venezuela"));
        list.add(new Country("Vanuatu"));
        return list;
    }
}

File: LoginPage/app/src/main/java/com/letrunghieu/loginpage/MainActivity.kt
package com.letrunghieu.loginpage

import android.content.Intent
import android.os.Build
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView
import androidx.annotation.RequiresApi

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val signUp = findViewById<TextView>(R.id.signUp)
        val singUpLayout = findViewById<View>(R.id.signUpLayout)
        val logIn = findViewById<TextView>(R.id.logIn)
        val logInLayout = findViewById<View>(R.id.logInLayout)
        val signIn = findViewById<View>(R.id.signIn)


        signUp.setOnClickListener {
            signUp.background = resources.getDrawable(R.drawable.switch_trks,null)
            signUp.setTextColor(resources.getColor(R.color.textColor,null))
            logIn.background = null
            singUpLayout.visibility = View.VISIBLE
            logInLayout.visibility = View.GONE
            logIn.setTextColor(resources.getColor(R.color.pinkColor,null))
        }
        logIn.setOnClickListener {
            signUp.background = null
            signUp.setTextColor(resources.getColor(R.color.pinkColor,null))
            logIn.background = resources.getDrawable(R.drawable.switch_trks,null)
            singUpLayout.visibility = View.GONE
            logInLayout.visibility = View.VISIBLE
            logIn.setTextColor(resources.getColor(R.color.textColor,null))
        }
        signIn.setOnClickListener {
            startActivity(Intent(this@MainActivity,LoginActivity::class.java))
        }
    }
}
